run/django/README.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with Django on Google Cloud Platform on Cloud Run (fully managed)
+
+[![Open in Cloud Shell][shell_img]][shell_link]
```

Comment:
It would, but it requires [more configuration](https://github.com/GoogleCloudPlatform/django-demo-app-unicodex/tree/latest/.gcloud). If you think it would be valuable to include, I can add it.

Reply:
Not necessary